mrp_food/models/product_supplierinfo.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2022 - Today: GRAP (http://www.grap.coop)
+# @author: Quentin DUPONT (quentin.dupont@grap.coop)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import _, api, fields, models
+
+from odoo.addons import decimal_precision as dp
+
+
+class SupplierInfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    product_standard_price = fields.Float(
+        string="Product actual standard price",
+        compute="_compute_product_standard_price",
```

Comment:
```suggestion
        related="product_tmpl_id.standard_price",
        store=True,
```

Edit : ah non, peut-être pas en fait, ``standard_price`` est un company_dependant de merde je crois...

Reply:
Ouep je tente tjrs le related mais là ça avait merdé